docs/_data/prague-2022-config.yaml

```diff
@@ -28,10 +28,10 @@ buttons:
     #   link: /schedule
     # - text: Buy a Ticket!
     #   link: /tickets
-    - text: Sponsor us
-      link: /sponsors/prospectus
-    #   - text: Submit a Talk
-    #     link: /cfp
+    #- text: Sponsor us
+    #  link: /sponsors/prospectus
+    - text: Submit a Talk
```

Comment:
@plaindocs what do you think about keeping two buttons for CFP + Sponsorship instead of the Welcome blog post + CFP?

Reply:
Also, the buttons at the top and bottom of the landing page aren't the same :)

Reply:
Isn't the point that they don't have to be?

Reply:
I don't remember that being the point ;) either way the welcome blog post is already linked at the "Update from the team" so I don't think we need it anymore, especially since we have a new blog post for the CFP

Reply:
why wait and risk forgetting about it amidst the website launch? :) the PR is open, and we already started getting sporsorship inquiries so I'd rather not lose the one button at the top of the page especially at the time of announcing the button next to it ;)